oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/manual/cache/CacheSlowAccessDispatcher.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.apache.skywalking.oap.server.core.analysis.manual.cache;
+
+import org.apache.skywalking.oap.server.core.analysis.SourceDispatcher;
+import org.apache.skywalking.oap.server.core.analysis.worker.TopNStreamProcessor;
+import org.apache.skywalking.oap.server.core.source.VirtualCacheSlowAccess;
+
+public class CacheSlowAccessDispatcher implements SourceDispatcher<VirtualCacheSlowAccess> {
+
+    @Override
+    public void dispatch(VirtualCacheSlowAccess source) {
+        if (source.getOp().equals("read")) {
+            TopNCacheReadCommand readCommand = new TopNCacheReadCommand();
+            readCommand.setId(source.getId());
+            readCommand.setCommand(source.getCommand() + " " + source.getKey());
+            readCommand.setLatency(source.getLatency());
+            readCommand.setTraceId(source.getTraceId());
+            readCommand.setServiceId(source.getCacheServiceId());
+            readCommand.setTimeBucket(source.getTimeBucket());
+            TopNStreamProcessor.getInstance().in(readCommand);
+        } else {
+            TopNCacheWriteCommand writeCommand = new TopNCacheWriteCommand();
```

Comment:
Do we have other commands? Should we use `else if` here?

Reply:
We create `VirtualCacheSlowAccess` only  when the op is write/read

Reply:
OK, but when you change it to an enum, I think doing a `else if` check should be super fast, and make codes better.